feature/tabsflow/home/src/main/java/com/epmedu/animeal/home/presentation/ui/map/RouteView.kt

```diff
@@ -130,15 +132,17 @@ private fun fetchRoute(
     mapboxNavigation: MapboxNavigation,
     onRouteResult: (result: RouteResult) -> Unit
 ) {
-    state.feedState.feedPoint?.coordinates?.let { feedingPointLocation ->
-        mapView.fetchRoute(
-            mapBoxRouteInitOptions = mapBoxRouteInitOptions,
-            navigation = mapboxNavigation,
-            path = MapPath(
-                state.locationState.location.toPoint(),
-                feedingPointLocation
-            ),
-            onRouteResult = onRouteResult
-        )
+    (state.locationState as? LocationState.ExactLocation)?.location?.toPoint()?.let { exactUserLocation ->
+        state.feedState.feedPoint?.coordinates?.let { feedingPointLocation ->
```

Comment:
Let's use 2 variables and `if` statement instead of 2 sequential `let`
```suggestion
        val exactUserLocation =
        (state.locationState as? LocationState.ExactLocation)?.location?.toPoint()
    val feedingPointLocation = state.feedState.feedPoint?.coordinates

    if (exactUserLocation != null && feedingPointLocation != null)
```

Reply:
Fair point, addressed with an additional commit